src/XIVLauncher.Common/Dalamud/DalamudInjectorArgs.cs

```diff
@@ -0,0 +1,22 @@
+namespace XIVLauncher.Common.Dalamud
+{
+    public static class DalamudInjectorArgs
+    {
+        public const string Launch = "launch";
```

Comment:
These consts don't match our style guide, but I think we should focus on fixing this later with the Great Goatcorp Lint Bonanza.